cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java

```diff
@@ -117,6 +117,9 @@ private boolean setRegulatingControlVoltage(String controlId,
                     .add();
         }
 
+        gen.setProperty("RegulatingControl", controlId);
+        gen.setProperty("targetDeadBand", String.valueOf(control.getTargetDeadBand()));
```

Comment:
Is it useful? If the `RegulatingControl` only regulates generators, we can put the targetDeadband as 0.

Reply:
We tried to keep the original value read from input data, even it is not applicable in IIDM for generators. Some regulating controls in conformity test cases have `targetDeadband != 0`.